docs/how-tos/train-deploy-ml.md

```diff
@@ -299,11 +299,25 @@ If no data appears after the sync interval, check the **LOGS** tab for errors.
 Your camera now saves images at the configured time interval.
 When training machine learning models, it is important to supply a variety of images.
 The dataset you create should represent the possible range of visual input.
-This may include capturing images of different angles, different configurations of objects and different lighting conditions
+This may include capturing images of different angles, different configurations of objects and different lighting conditions.
 The more varied the provided dataset, the more accurate the resulting model becomes.
 
 Capture at least 10 images of anything you want your machine to recognize.
 
+{{< expand "For more tips and tricks on improving model accuracy, click here." >}}
+
+- **More data means better models:** Incorporate more data from varying environments, such as lighting conditions or distances, to improve your model’s overall performance.
```

Comment:
```suggestion
- **More data means better models:** Incorporate as much data as you practically can to improve your model’s overall performance.
```
The "varying environments" bit is covered in both the paragraph above as well as the points below, so this one can emphasize quantity

Reply:
true. thanks!